src/Airship/Resource.hs

```diff
@@ -125,7 +116,6 @@ defaultResource = Resource { allowMissingPost       = return False
                            , implemented            = return True
                            , isAuthorized           = return True
                            , isConflict             = return False
-                           , knownContentType       = return True
```

Comment:
See the last commit - I suspect there are kinds of checks that might be down on `Accept` that aren't possible if just using `contentTypesProvided`.